### DISCUSSION TITLE
在网关重载后对所有节点进行测速好像未生效

### DISCUSSION THREAD

**Discussion by TwoBall** [kkkgo/PaoPaoGateWay#15] [问和答]
只在更改ppgw.ini之后第一次载入会测速，之后重启就不会测速了，要手动进web界面选择一次，默认是Direct

**Comment by TwoBall** [comment]
![1.jpg](https://img1.imgtp.com/2023/07/20/EZMgfB8c.jpg)
![2.jpg](https://img1.imgtp.com/2023/07/20/58Bg09yZ.jpg)

**Comment by TwoBall** [comment]
不管是yaml模式还是suburl模式都是如此

**Comment by kkkgo** [comment]
是在加载iso启动之后，没有进行测速选节点吗？只有更改ppgw.ini配置才触发测速？

**Reply by TwoBall** [reply]
是的，只有在更新产生新的节点配置的时候才会进行一次测速，再重启iso就不会测速了

**Comment by kkkgo** [comment]
现在确认了一个问题，7月份新引入的功能导致了bug。稍后修复。

**Reply by TwoBall** [reply]
好的，这个问题主要是有时候停电了无法自动恢复，其余都不影响

**Comment by kkkgo** [comment]
使用最新release试试。https://github.com/kkkgo/PaoPaoGateWay/releases
